configs/src/deployments/json/modeMainnet.json

```diff
@@ -0,0 +1,351 @@
+      "blockNumber": 13627035,
+      "deploymentTx": "0x6f2584860a47cba596820e38895944a2d16191ba878312dd7ad19b5f6ef0baa2"
+    },
+    "PluginRepoBase": {
+      "address": "0x2eD011bBbb1B709265C498620D2F4116aB057fC3",
+      "blockNumber": 21117127,
+      "deploymentTx": "0xb74e7360ac33cdc371d7e97a41fd7e1e53c5b3a009b5d683b8f1e62617c89b22"
+    },
+    "PluginRepoFactory": {
+      "address": "0x78E839210aa3072ce638A2d6b08DA611ca361A0E",
+      "blockNumber": 21117127,
+      "deploymentTx": "0xb74e7360ac33cdc371d7e97a41fd7e1e53c5b3a009b5d683b8f1e62617c89b22"
+    },
+    "PluginRepoRegistryProxy": {
+      "address": "0xCe0B4124dea6105bfB85fB4461c4D39f360E9ef3",
+      "blockNumber": 13626971,
+      "deploymentTx": "0x55bffedf857bacc381665627598fe1c551f2bb94d7fa5763bb203f2f7c8b9411"
+    },
+    "PluginRepoRegistryImplementation": {
+      "address": "0xeB98a71d69a1e12B62c10368D9dA5364CE0f7178",
+      "blockNumber": 21117134,
+      "deploymentTx": "0x3f89d4158c23f62cee999f6bdab0dce14307482307df81be0617b09bcf73b683"
+    },
+    "PluginSetupProcessor": {
+      "address": "0x308a1DC5020c4B5d992F5543a7236c465997fecB",
+      "blockNumber": 13626979,
+      "deploymentTx": "0x0abcba1188e008073625bd86bfc2d8538caf6d0156ca136a360c3b686f67384d"
+    },
+    "PluginENSSubdomainRegistrarProxy": {
+      "address": "0x6115b7F05cCeF2D883AB64c88ba7e4BD1b877215",
+      "blockNumber": 13626950,
+      "deploymentTx": "0xc912dedb6d916e03253c5339b5deae59c49bcbaafc6bda8d3855aba45ce95a18"
+    },
+    "PluginENSSubdomainRegistrarImplementation": {
+      "address": "0x74b3B3504B5d6D1c6247009c9b1e3D8cFF7bd445",
+      "blockNumber": 13626947,
+      "deploymentTx": "0x239f520cedc7cf6511bdba16995207ed3ffebfe38bce1fa8168ae18ddf6fe222"
+    },
+    "PublicResolver": {
+      "address": "0xc4ddBd0472C2688F5d278dAFC02E66EE176C8DE3",
+      "blockNumber": 13626902,
+      "deploymentTx": "0xfe5a86cb85f9c0c4dd53fe5dec2a4c67ec0bf29d447f9e2d81e86413cd73b295"
+    },
+    "TokenVotingSetup": {
+      "address": "0x9bD1EF0C495F6b078d9A82586A35F4d623384E97",
+      "blockNumber": 21115881,
+      "deploymentTx": "0x67db465333f14eb35fca00358d12773672ffe39e7fdc6828829abd825d5ca66b"
+    },
+    "TokenVotingSetupImplementation": {
+      "address": "0x9EB3C2F3076EE42F6B3cDf6d9407532E29b2FA58",
+      "blockNumber": 21115881,
+      "deploymentTx": "0x67db465333f14eb35fca00358d12773672ffe39e7fdc6828829abd825d5ca66b"
+    },
+    "AdminRepoProxy": {
+      "address": "0x326A2aee6A8eE78D79E7E956DE60C6E452f76a8e",
+      "blockNumber": 13627066,
+      "deploymentTx": "0x3a633c367c2f4ed2911570377d37a5631bb7e6fad058e0f55140d3210f108c24"
+    },
+    "ManagementDAOProxy": {
+      "address": "0xF3AaA3372EbBf01b923a4Cc98Cd847126b3D73cA",
+      "blockNumber": 13626882,
+      "deploymentTx": "0xa25407d3c0fd9db57545a65f70f25eefd60ee1fcc161038539d67662f8dcf53f"
+    },
+    "ManagementDAOImplementation": {
+      "address": "0x4C8288220e7Bf31e6dEe16fB58d652E024529682",
```

Comment:
ManagementDAOProxy is pointing to `0x741061B01d3A95a7734B594f1884b6c30E0fDA96`, which doesn't coincide with DaoBase either.

Reply:
https://github.com/aragon/osx-commons/pull/147#discussion_r2012197633